.travis.yml

```diff
@@ -68,6 +66,18 @@ env:
     - SCRIPT="cd play-scala-tls-example            && scripts/test-sbt" TRAVIS_JDK=11
     - SCRIPT="cd play-scala-websocket-example      && scripts/test-sbt" TRAVIS_JDK=11
 
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    # The gRPC samples fail on 2.8.x because of a non-evicted dependency of akka-http causing 
+    # cross-version checks in Akka HTTP to fail.
+    - env: SCRIPT="cd play-java-grpc-example            && scripts/test-sbt"
+    - env: SCRIPT="cd play-scala-grpc-example           && scripts/test-sbt"
+    - env: SCRIPT="cd play-java-grpc-example            && scripts/test-sbt" TRAVIS_JDK=11
+    - env: SCRIPT="cd play-scala-grpc-example           && scripts/test-sbt" TRAVIS_JDK=11
+
```

Comment:
Raised https://github.com/playframework/play-samples/issues/105 (related to #102)